tools/mob_suite/mob_recon.xml

```diff
@@ -1,7 +1,8 @@
-<tool id="mob_recon" name="MOB-Recon" version="2.0.5+galaxy0">
+
```

Comment:
Delete new line here.

Reply:
Yes, will remove extra new line. Thank you